entrypoint.sh

```diff
@@ -5,137 +5,121 @@ set -o pipefail
 set -o xtrace
 
 # Require environment variables.
-if [ -z "${SUBSPACE_HTTP_HOST-}" ] ; then
-    echo "Environment variable SUBSPACE_HTTP_HOST required. Exiting."
-    exit 1
+if [ -z "${SUBSPACE_HTTP_HOST-}" ]; then
+  echo "Environment variable SUBSPACE_HTTP_HOST required. Exiting."
+  exit 1
 fi
 # Optional environment variables.
-if [ -z "${SUBSPACE_BACKLINK-}" ] ; then
-    export SUBSPACE_BACKLINK=""
+if [ -z "${SUBSPACE_BACKLINK-}" ]; then
+  export SUBSPACE_BACKLINK=""
 fi
 
-if [ -z "${SUBSPACE_IPV4_POOL-}" ] ; then
-    export SUBSPACE_IPV4_POOL="10.99.97.0/24"
+if [ -z "${SUBSPACE_IPV4_POOL-}" ]; then
+  export SUBSPACE_IPV4_POOL="10.99.97.0/24"
 fi
-if [ -z "${SUBSPACE_IPV6_POOL-}" ] ; then
-    export SUBSPACE_IPV6_POOL="fd00::10:97:0/112"
+if [ -z "${SUBSPACE_IPV6_POOL-}" ]; then
+  export SUBSPACE_IPV6_POOL="fd00::10:97:0/112"
 fi
-if [ -z "${SUBSPACE_NAMESERVER-}" ] ; then
-    export SUBSPACE_NAMESERVER="1.1.1.1"
+if [ -z "${SUBSPACE_NAMESERVER-}" ]; then
+  export SUBSPACE_NAMESERVER="1.1.1.1"
 fi
 
-if [ -z "${SUBSPACE_LETSENCRYPT-}" ] ; then
-    export SUBSPACE_LETSENCRYPT="true"
+if [ -z "${SUBSPACE_LETSENCRYPT-}" ]; then
+  export SUBSPACE_LETSENCRYPT="true"
 fi
 
-if [ -z "${SUBSPACE_HTTP_ADDR-}" ] ; then
-    export SUBSPACE_HTTP_ADDR=":80"
+if [ -z "${SUBSPACE_HTTP_ADDR-}" ]; then
+  export SUBSPACE_HTTP_ADDR=":80"
 fi
 
-if [ -z "${SUBSPACE_LISTENPORT-}" ] ; then
-    export SUBSPACE_LISTENPORT="51820"
+if [ -z "${SUBSPACE_LISTENPORT-}" ]; then
+  export SUBSPACE_LISTENPORT="51820"
 fi
 
-if [ -z "${SUBSPACE_HTTP_INSECURE-}" ] ; then
-    export SUBSPACE_HTTP_INSECURE="false"
+if [ -z "${SUBSPACE_HTTP_INSECURE-}" ]; then
+  export SUBSPACE_HTTP_INSECURE="false"
 fi
 
 export DEBIAN_FRONTEND="noninteractive"
 
-if [ -z "${SUBSPACE_IPV4_GW-}" ] ; then
-        export SUBSPACE_IPV4_PREF=$(echo ${SUBSPACE_IPV4_POOL-} | cut -d '/' -f1 |sed 's/.0$/./g' )
-	export SUBSPACE_IPV4_GW=$(echo ${SUBSPACE_IPV4_PREF-}1)
+if [ -z "${SUBSPACE_IPV4_GW-}" ]; then
+  export SUBSPACE_IPV4_PREF=$(echo ${SUBSPACE_IPV4_POOL-} | cut -d '/' -f1 | sed 's/.0$/./g')
```

Comment:
As this env variable is also used at https://github.com/subspacecommunity/subspace/blob/master/handlers.go#L374, can it be exported even when $SUBSPACE_IPV4_GW is supplied by user? or add the env variable to docker run parameter and docker-compose parameter so users will know it when changing the default subnet

Reply:
Think you meant https://github.com/subspacecommunity/subspace/blob/master/handlers.go#L378

Raise a bug for this, as this PR is to address other issues within the project